providers/base/units/graphics/jobs.pxu

```diff
@@ -395,14 +399,17 @@ requires:
 command:
  glxgears
  true
-_summary: Test that glxgears works for current video card
-_description:
- PURPOSE:
+summary:
+    Test that glxgears works for current video card
+description:
+    Run glxgears and as the tester to validate the outcome.
+    Test will cover all devices without integrated display, such as Desktop.
+purpose:
      This test tests the basic 3D capabilities of your current video card
```

Comment:
The `description` field is not required when there is a `purpose` field, so I suggest to merge both:

```suggestion
purpose:
     Tests the basic 3D capabilities of your current video card. This test covers all devices without an integrated display, such as desktops.
```